CMakeLists.txt

```diff
@@ -31,9 +31,9 @@ install(FILES "${RecastNavigation_BINARY_DIR}/version.h" DESTINATION
 # Needed for recastnavigation.pc.in
 set(prefix ${CMAKE_INSTALL_PREFIX})
 set(exec_prefix "\${prefix}")
```

Comment:
What do you think about omitting `exec_prefix` from the `.pc.in` altogether?

Reply:
I don't have strong opinions here but I'm all for simplifying the build pipeline